api/scpca_portal/test/test_metadata_file.py

```diff
@@ -76,6 +76,17 @@ def test_write_metadata_dicts_read_write_values_match(self):
             output_list_of_dicts = list(csv.DictReader(output_file, delimiter=common.TAB))
             self.assertEqual(self.dummy_list_of_dicts, output_list_of_dicts)
 
+    def test_write_metadata_dicts_read_write_no_empty_values(self):
+        field_name = "capital"
+        item = 3
+        metadata_file.write_metadata_dicts(
+            self.dummy_list_of_dicts, self.dummy_output_path, fieldnames=self.dummy_field_names
+        )
+
+        with open(self.dummy_output_path) as output_file:
+            output_list_of_dicts = list(csv.DictReader(output_file, delimiter=common.TAB))
+            self.assertEqual(common.NA, output_list_of_dicts[item][field_name])
```

Comment:
```suggestion
    def test_write_metadata_dicts_read_write_no_empty_values(self):
         destinations = [
            {
                "scpca_project_id": "SCPCP999993",
                "scpca_sample_id": "SCPCS999993",
                "scpca_library_id": "SCPCL999993",
                "country": "Antarctica",
                "language": "Antarctic English",
                "empty": None,
            },
        ]

        metadata_file.write_metadata_dicts(destinations, self.dummy_output_path)

        with open(self.dummy_output_path) as output_file:
            for destination in list(csv.DictReader(output_file, delimiter=common.TAB))
                        self.assertEqual(common.NA, destination["empty"])
```

Reply:
See the commit at 2902d82e4a1e0fe3f3f476782a02af754698b2f2.